Source/WebCore/platform/graphics/avfoundation/objc/SourceBufferParserAVFObjC.mm

```diff
@@ -232,12 +231,14 @@ AtomString codec() const override
 
 SourceBufferParserAVFObjC::~SourceBufferParserAVFObjC()
 {
+    ALWAYS_LOG_IF_POSSIBLE(LOGIDENTIFIER);
```

Comment:
why log only the destructor?

Reply:
We don't have access to a logger in the constructor, it will never log, so instead we log in setLogger